subsys_display_view.py

```diff
@@ -58,13 +58,13 @@ def stop(cls):
         pass
 
     @classmethod
-    def run(cls, frame: numpy.ndarray, **kwargs):
+    def run(cls, frame: numpy.ndarray, variables_dict: dict):
```

Comment:
This **kwargs has some hided functionality, it means you're able to throw in that function an undefined number of keyword/value pairs and it will automatically construct a dictionary with all those passed parameters.
It was probably unused and thus unnecessary, but i'm just ensuring nothing was missed here an that no other call elsewhere was using that functionality ^.^